From 75584fe660880b332fbf60dd7968e2ed8b49a38b Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Fri, 20 Dec 2019 16:12:50 +0100
Subject: [PATCH] type_compat: Add missing __s64 and __u64 definitions for
 non-Linux

Just for the case without Linux-compatible definitions.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 include/sound/type_compat.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/sound/type_compat.h b/include/sound/type_compat.h
index d4790c1f..5b765f44 100644
--- a/include/sound/type_compat.h
+++ b/include/sound/type_compat.h
@@ -9,9 +9,11 @@
 typedef uint8_t __u8;
 typedef uint16_t __u16;
 typedef uint32_t __u32;
+typedef uint64_t __u64;
 typedef int8_t __s8;
 typedef int16_t __s16;
 typedef int32_t __s32;
+typedef int64_t __s64;
 
 #include <endian.h>
 #include <byteswap.h>
